modules/client-configuration/pages/cobbler.adoc

```diff
@@ -3,6 +3,13 @@
 
 Cobbler is an installation server that allows you to perform unattended system installations.
 It can be used on server, client, or guest systems, and in virtual environments.
+On {productname} Cobbler is installed on the server.
+
+[IMPORTANT]
+====
+{suse} only supports Cobbler functions that are available in the {productname} {webui}, or through the {productname} API.
+All features documented here are supported.
```

Comment:
I wonder if it works better to turn this around:

```suggestion
Only supported features are documented here.
```

The most important info is presumably the supported features, rather than the documented features?

Reply:
yes, much better, IMO.